### ISSUE TITLE
Datasets with reconstructions and fuzzy mappings

### ISSUE THREAD

**Issue by FredericBlum** [lexibank/lexibank-analysed#62]
@LinguList @chrzyki We have three Pano-Tacanan datasets (girardprototakanan, oliveiraprotopanoan, valzarpanotakana) that have reconstructions provided by the original author. The problem is, however, that the authors (and we as well) have adopted the reconstructed semantics for all the terms that are part of the cognate set. With @MuffinLinwist being occupied with his thesis and me with mine, a detailed re-mapping of the individual entries is not very likely. We are talking 3.000 different concepts here, all under the umbrella of ~500 reconstructed (and manually mapped) concepts.

How should we proceed with those datasets?

a) Exclude from the release
b) Exclude from LexCore, but keep in CogCore
c) Only exclude from ClicsCore

**Comment by FredericBlum** [comment]
The `blumpanotacana` dataset will provide a reasonable coverage of the languages involved, but is without reconstructions. So my proposal would probably be b), so we can explicitly present the data as CogCore, but alert people about not using those for other stuff that is based on concept comparisons.

**Comment by LinguList** [comment]
So, we have a situation where you have one cognate set and one meaning mapped for all of them, although individual meanings diverge?

**Comment by LinguList** [comment]
Or do we talk about something different here? If it is one meaning for one cognate set, my traditional approach would have been to put these at the side and also to not necessarily link the concepts to Concepticon.

**Comment by FredericBlum** [comment]
Summarizing the discussion with @LinguList yesterday, one possibility is to include them as CogCore and ProtoCore, but not in LexCore. Additionally, we will use a disclaimer in the Notes.md. There might be other datasets (e.g. viegasbarroschaco) with similar situations.

**Comment by LinguList** [comment]
For ViegasBarrosChaco, I am not quite sure, as I now answered in the repository, but we should probably have a look to what degree we might want to have a collection of repos that are reconstruction only, without much wordlist underneath it. There's also the dataset lexiank/hruschkaturkic, which I prepared from the info from the authors, but it is even more extreme. 

If we say that Lexibank is first and all a repository that lists Wordlists, we could say that any dataset that does not fall into this category of Edelwordlist, can be listed, but would not be propagated in the "best off". 

The CLLD landing page anyway only includes those datasets that provide data that we can process (searching for colexifications, searching for sound inventories, or searching for cognates). So even if we INCLUDE these datasets now, they would fall beyond the radar, since the CLLD website in the end only includes data with more than 200 concepts filled by valid CLTS forms now.

**Comment by LinguList** [comment]
A question that we should discuss with @xrotwang is to which degree we can make the CLLD app also share the inferred features (mama / papa, etc.). I think this would be beautiful, but cannot tell how difficult this would be for us.

**Comment by xrotwang** [comment]
> A question that we should discuss with @xrotwang is to which degree we can make the CLLD app also share the inferred features (mama / papa, etc.). I think this would be beautiful, but cannot tell how difficult this would be to implement it.

I guess this could look like https://hindukush.clld.org/ - which also ingests data from two CLDF datasets.

**Comment by LinguList** [comment]
I'd consider this as a beautiful next step :-)

**Comment by FredericBlum** [comment]
The inferred features could probably be combined with the upcoming discussion in #61 ? Would be really nice to have this.

Regarding the datasets: How would we handle those cases in the Lexibank sqlite, which we use as the main resource? Exclude them from the CLDF creation, by some extra tag?

There seem to be at least five such datasets (valzar, oliveira, girard, hruschkaturkic, possibly Holmie, possibly chacontukanoan).

**Comment by LinguList** [comment]
I have a very concrete idea of how to deal with data we want to ignore in one run, and we must modify my lexibank implementation here!

**Comment by LinguList** [comment]
My idea is to introduce a new column in the FormTable. This column could be called "Lexibank_Forms" or whatever, and it would only take those cases into account where we have, e.g.

* the language variety among many languages with the same glottocode that has most forms mapped to concepticon with valid CLTS representation

Right now, we KICK these data out in the `forms.csv` file. But we do not need to do that if we say we include EVERYTHING but FLAG afterwards. We could even think about different status codes. E.g., 1 for lexibank-analysed proper, 2 for lexibank proto, etc. Having this information in the FormTable won't really grow the data a lot, but it will give us MUCH more flexibility in the future. Even the clld app could then give ALL data, but allow users to select specific values in the column.

**Comment by LinguList** [comment]
Question: how to call the column? For consistency, it would be good to find a good name here. LexemeFilter (?). LexemeType? DataType?

**Comment by xrotwang** [comment]
Hm, the `parameterReference` in `FormTable` has pretty clear semantics. So, for the case of forms in a cognate set which are all mapped to the concept of the reconstructed form I would want to leave this column empty - except for the reconstructed form.

This does not mean any loss of information, because the mapping to the reconstructed concept can still be inferred from the cognateset membership. And it does not introduce a new kind meaning for `parameterReference`.

**Comment by xrotwang** [comment]
The semantics of `parameterReference = NULL` is much more transparent and easier to deal with than "mapped to some concept which might be correct, but really not sure".

**Comment by LinguList** [comment]
Yes. I agree. Linking to a potentially false concept should not be done. 

The selection of individual doculects over others representing the same glottocode, however, is something else, right? So instead of only selecting ONE doculect for stan1295 for inclusion in forms.csv, I'd propose to take ALL, but indicate which is the preferred one by adding a column that allows us to retrieve it later.

**Comment by LinguList** [comment]
I would suggest to close this issue now, since I consider it solved: we leave out all datasets that are "etymological" or not explicitly concept-based (i.e., onomasiologically oriented). 

An example, why this is important, is the new version of [DIACL](https://diacl.uni-frankfurt.de/), where I checked German, just out of curiosity. There, they have all kinds of definitions for German words, they look "onomasiological", since they have the language-concept-form structure, but they are mere glosses for individual senses that are often even not exact or even completely wrong.

Check German [(ver)zehren](https://diacl.uni-frankfurt.de/Lexeme/Details/86385): it is glossed as "to tear up, destroy" in the database, since they found this in some sources, but *verzehren* does not have this as primary sense. The primary sense is "to eat", and "zehren" is old-fashioned, so already the brackets are problematic, etc. 

Thus, any resource that glosses "on the fly" as they tend to do in etymological dictionaries, is not apt for the kind of data presented in lexibank.
